Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2252,9 +2274,8 @@ private void HandleProximityScaling()
                 }
 
                 // Get the max radius possible of our current bounds plus the proximity
-                float maxRadius = Mathf.Max(Mathf.Max(currentBoundsExtents.x, currentBoundsExtents.y), currentBoundsExtents.z);
-                maxRadius *= maxRadius;
-                maxRadius += handleCloseProximity + handleMediumProximity;
+                float maxRadius = currentBoundsExtents.sqrMagnitude;
+                maxRadius += 3 * handleMediumProximity * handleMediumProximity;
```

Comment:
can you explain why this is now squared length and triple medium proximity  added?

Reply:
without a comment this seems pretty arbitrary just to make it work?

Reply:
I was copying what we did in bounds control 

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/18fe32fd8d627216754885c7fecf4cf2434aa07d/Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/ProximityEffect/ProximityEffect.cs#L153

Reply:
ha! i'm glad i left that comment there - could you please copy over the comment from bounds control two lines above so it's more clear what this does :D